From a7901c0f7c748afd75329a20c282f9c6cb7eaa7b Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Fri, 6 Dec 2019 10:41:10 +0700
Subject: [PATCH 330/334] arm64: dts: renesas: r8a774a1: add misisng enabled
 temperature for IPA

This patch adds missing enabled temperature for IPA.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774a1.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774a1.dtsi b/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
index c30ed5f9ee69..6aba5cb350b4 100644
--- a/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
@@ -2670,6 +2670,12 @@
 			sustainable-power = <3874>;
 
 			trips {
+				threshold: trip-point0 {
+					temperature = <90000>;
+					hysteresis = <1000>;
+					type = "passive";
+				};
+
 				target: trip-point1 {
 					temperature = <100000>;
 					hysteresis = <1000>;
-- 
2.17.1

